src/layouts/layout.jsx

```diff
@@ -54,34 +59,39 @@ export const StyledFooterText = styled.label`
   line-height: 38px;    
 `
 
-function Layout({ children }) {
+function Layout({ children }) {  
+  const [contextState, setContextState] = useState(initialState);
+
   return (
     <>
-      <div className={styles.bannerBlogContainer}>
-        <StyledContainerNavBarXL>
-          <StyledContainerHeader>
-            <div className={styles.navMenuContainer}>
-              <Link
-                to="/"
-                id="logo-xl">
-                <StaticImage 
-                  src="../../static/images/logo.svg"
-                  alt=""
-                  width={56}
-                  height={56}
-                />
-              </Link>
-              <NavMenu />
-            </div>
-            <StyledGetStartedButton id="header-getintouch" 
-              href="#/" 
-            >
-            <StyledGetStartedTextButton>GET STARTED</StyledGetStartedTextButton>
-            </StyledGetStartedButton>
-          </StyledContainerHeader>
-        </StyledContainerNavBarXL>
-        {children}
-      </div>
+      <AppContext.Provider value={{ state: contextState, setState: setContextState }}>
+        <div className={classnames(styles[`${contextState}Banner`])}>
```

Comment:
You don't need `classnames`
```suggestion
        <div className={styles[`${contextState}Banner`]}>
```

Reply:
Done